server/src/main/java/org/apache/druid/client/HttpServerInventoryView.java

```diff
@@ -132,7 +129,7 @@ public HttpServerInventoryView(
 
 
   @LifecycleStart
-  public void start() throws Exception
+  public void start() throws ISE
```

Comment:
Looks like `ISE` is not thrown?

Reply:
It may be thrown, here is the code:
```java
if (!lifecycleLock.canStart()) {
        throw new ISE("can't start.");
}
```

Reply:
Ah, ok. ISE is a RuntimeException and not a checked exception. I think it's fine to remove it.

Reply:
Done.